Microsoft.Toolkit.Uwp.SampleApp/SamplePages/AdaptiveGridView/AdaptiveGridViewPage.xaml.cs

```diff
@@ -42,14 +43,14 @@ public AdaptiveGridViewPage()
             AdaptiveGridViewControl.SelectionChanged += AdaptiveGridViewControl_SelectionChanged;
         }
 
-        private void AdaptiveGridViewControl_SelectionChanged(object sender, Windows.UI.Xaml.Controls.SelectionChangedEventArgs e)
+        private void AdaptiveGridViewControl_SelectionChanged(object sender, SelectionChangedEventArgs e)
         {
             SelectedItemCountTextBlock.Text = AdaptiveGridViewControl.SelectedItems.Any()
                 ? $"You have selected {AdaptiveGridViewControl.SelectedItems.Count} items."
                 : "You haven't selected any items";
         }
 
-        private async void AdaptiveGridViewControl_ItemClick(object sender, Windows.UI.Xaml.Controls.ItemClickEventArgs e)
+        private async void AdaptiveGridViewControl_ItemClick(object sender, ItemClickEventArgs e)
```

Comment:
For me this is throwing errors :S

![image](https://cloud.githubusercontent.com/assets/1771785/20233371/be7168ae-a823-11e6-9cff-cd2f5b09267f.png)